flutter/android/app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,9 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.READ_FRAME_BUFFER" />
+    <uses-permission android:name="android.permission.CAPTURE_VIDEO_OUTPUT" />
+    <uses-permission android:name="android.permission.CAPTURE_SECURE_VIDEO_OUTPUT" />
```

Comment:
Curiously，these permission is for what? Are you testing new screen capture method on Android?

Reply:
Yes

Reply:
Please let us know if you have better solution.

Reply:
Can I know why do you wanna change?

Reply:
I wanted to get rid of android screencast popup, but adding these permissions in basis works, with rustdesk signed with system key, i just want to test it before i do something on main repo